internal/service/connect/routing_profile.go

```diff
@@ -76,7 +76,7 @@ func ResourceRoutingProfile() *schema.Resource {
 				Type:     schema.TypeSet,
 				Optional: true,
 				MinItems: 1,
-				MaxItems: 10,
+				MaxItems: 50,
```

Comment:
From the documentation for [Amazon Connect service quotas](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-service-limits.html), it mentions that `Queues per routing profile per instance` is an `Adjustable` quota. Hence, the `MaxItems` check can be removed entirely.

```suggestion